recirq/quantum_chess/dynamic_look_ahead_heuristic_circuit_transformer_test.py

```diff
@@ -0,0 +1,227 @@
+# Copyright 2020 Google
```

Comment:
these tests were written as sanity checks, and i'm thinking they might be useful to keep around for debugging until after the swap update algorithm is implemented and proper tests for circuit transformations can be written?

Reply:
I think we should probably keep these tests, but maybe split out DynamicLookAheadHeuristicCircuitTransformer so that it matches the name of the test file.

Reply:
ack, i refactored the initial mapping logic out to `initial_mapping_utils.py` and renamed this to `initial_mapping_utils_test.py`